Gemfile

```diff
@@ -109,6 +108,10 @@ gem 'strong_migrations'
 # Rails console colours
 gem 'colorize'
 
+group :production do
```

Comment:
should anything change on the build setup for bundle to include this group?

Reply:
I don't think so, it looks at RAILS_ENV. From [docs](https://bundler.io/v2.2/guides/groups.html):

```
By default, a Rails generated app calls Bundler.require(:default, Rails.env) in your application.rb, which links the groups in your Gemfile to the Rails environment.
```

Reply:
It worked locally with RAILS_ENV=production

Reply:
we'll have to set RAILS_ENV before bundle install stage ?
https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/Dockerfile#L39

Reply:
or pass `--with production` — I'll add

Reply:
added